src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Instead of doing this, what do you think of not using a mock for `SconeSessionSecurityConfig` and to use a real instance in tests ?